etcdserver/raft.go

```diff
@@ -226,6 +226,10 @@ func (r *raftNode) start(s *EtcdServer) {
 				if err := r.storage.Save(rd.HardState, rd.Entries); err != nil {
 					plog.Fatalf("raft save state and entries error: %v", err)
 				}
+				if !raft.IsEmptyHardState(rd.HardState) {
+					proposalsCommitted.Set(float64(rd.HardState.Commit))
```

Comment:
@brian-brazil Here is the problem, for leader we can know the increment. for followers, this value can be bumped (due to receive a snapshot) suddenly. it is just inconvenient to modify this metrics at multiple places.